hazelcast-client/src/main/java/com/hazelcast/client/spi/impl/ClientInvocation.java

```diff
@@ -171,6 +172,7 @@ public void notify(ClientMessage clientMessage) {
         clientInvocationFuture.complete(clientMessage);
     }
 
+    @SuppressWarnings({ "checkstyle:cyclomaticcomplexity", "checkstyle:npathcomplexity" })
```

Comment:
The workaround would be to simplify the method, so less branches etc. (normally you can achieve this by pulling out methods).